docs/cloud/satellites/managing.md

```diff
@@ -98,22 +98,87 @@ Currently selected: No
 
 ### Clearing cache
 
-To clear the cache of a satellite, run the following while a satellite is selected:
+The easiest way to clear cache on a satellite is to restart it using the `update` command with `--drop-cache` flag.
+
+```bash
+earthly satellite update --drop-cache my-satellte
+```
+
+Note that this operation can take a while, and the satellite may also receive any available updates during the process.
+
+The `earthly prune` command can also be used. In many cases, however, it takes longer than using `satellite update`.
+The prune command also requires the satellite to be selected before running.
 
 ```bash
 earthly prune -a
 ```
 
-### Upgrading a satellite
+### Updating a satellite
 
-Currently, satellites do not have an auto-update mechanism built in. In order to get a newer version of a satellite, you need to manually remove and re-launch the satellite. Note that this operation resets the cache.
+By default satellites will automatically update as new versions become available. 
```

Comment:
I would combine these two sentences:

Satellites receive version updates by default, unless they are pinned to a specific version (by using the --some-flag-here). Pinned versions will still receive minor patches, such as security updates.